assets/js/modules/tagmanager/components/common/ExistingTagNotice.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Tag Manager Existing Tag Notice component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { sprintf, __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../../datastore/constants';
+const { useSelect } = Data;
+
+export default function ExistingTagNotice() {
```

Comment:
I've combined them for now by moving `ExistingTagNotice` inside `FormInstructions` but kept the component for consistency with the other modules as it may still be useful by itself at some point.